asdf/schema.py

```diff
@@ -346,23 +346,12 @@ def _load_draft4_metaschema():
 
 @lru_cache()
 def load_custom_schema(url):
-    # Avoid circular import
-    from .tags.core import AsdfObject
-    custom = load_schema(url, resolve_local_refs=True)
-    core = load_schema(AsdfObject.yaml_tag)
-
-    def update(d, u):
```

Comment:
I don't think it's actually necessary to merge the core schema with the custom schema, since we already check against the core schema in a first validation pass.